src/preload.js

```diff
@@ -88,96 +28,98 @@ const preload_character_trials = {
+        mediaAssets.video.pseudoIntroEn,
+        mediaAssets.video.pseudoPost2BlockEn,
+        mediaAssets.video.pseudoPostPracticeEn,
+        mediaAssets.video.pseudoRewardAnimation1En,
+        mediaAssets.video.pseudoRewardAnimation2En,
       ],
     };
   } else {
     videoBlocks = {
       1: [
-        latinBlock1En,
-        latinEndEn,
-        latinIntroEn,
-        latinPost2BlockEn,
-        latinPostPracticeEn,
-        latinRewardAnimation1En,
-        latinRewardAnimation2En,
+        mediaAssets.video.latinBlock1En,
+        mediaAssets.video.latinEndEn,
+        mediaAssets.video.latinIntroEn,
+        mediaAssets.video.latinPost2BlockEn,
+        mediaAssets.video.latinPostPracticeEn,
+        mediaAssets.video.latinRewardAnimation1En,
+        mediaAssets.video.latinRewardAnimation2En,
       ],
     };
   }
 } else {
   // eslint-disable-next-line no-lonely-if
-  if (config.pseudoFont) {
+  if (config().pseudoFont) {
     videoBlocks = {
       1: [
-        pseudoBlock1Es,
-        pseudoEndEs,
-        pseudoIntroEs,
-        pseudoPost2BlockEs,
-        pseudoPostPracticeEs,
-        pseudoRewardAnimation1Es,
-        pseudoRewardAnimation2Es,
+        mediaAssets.video.pseudoBlock1Es,
+        mediaAssets.video.pseudoEndEs,
+        mediaAssets.video.pseudoIntroEs,
+        mediaAssets.video.pseudoPost2BlockEs,
+        mediaAssets.video.pseudoPostPracticeEs,
+        mediaAssets.video.pseudoRewardAnimation1Es,
+        mediaAssets.video.pseudoRewardAnimation2Es,
       ],
     };
   } else {
     videoBlocks = {
       1: [
-        latinBlock1Es,
-        latinEndEs,
-        latinIntroEs,
-        latinPost2BlockEs,
-        latinPostPracticeEs,
-        latinRewardAnimation1Es,
-        latinRewardAnimation2Es,
+        mediaAssets.video.latinBlock1Es,
+        mediaAssets.video.latinEndEs,
+        mediaAssets.video.latinIntroEs,
+        mediaAssets.video.latinPost2BlockEs,
+        mediaAssets.video.latinPostPracticeEs,
+        mediaAssets.video.latinRewardAnimation1Es,
+        mediaAssets.video.latinRewardAnimation2Es,
       ],
     };
   }
 }
 
 const imageBlocks = {
   2: [
-    star,
-    advance,
+    mediaAssets.images.star,
+    mediaAssets.images.advance,
+    mediaAssets.images.gradeKeyboard,
   ],
 };
 
 const audioBlocks = {
   3: [
-    feedbackCorrect,
-    feedbackIncorrect,
+    mediaAssets.audio.feedbackCorrect,
+    mediaAssets.audio.feedbackIncorrect,
   ],
 };
 
 // Automatically populate the audioContent object with the audio files
-export const imgContent = preloadObj2contentObj(imageBlocks);
-export const videoContent = preloadObj2contentObj(videoBlocks);
-export const audioContent = preloadObj2contentObj(audioBlocks);
+export const imgContent = imageBlocks;
+export const videoContent = videoBlocks;
+export const audioContent = audioBlocks;
 
 const preload_video_trials = Object.entries(videoBlocks).map(([idx, img_block]) => ({
   type: jsPsychPreload,
```

Comment:
We shouldn't need this file anymore. All preloading is handled by the createPreloadTrials helper function.

Reply:
I'm unable to delete this file for some reason, the problem is that util.js calls upon the preload function specifically for the mediaassets. If I try to instead import the assets from experiment.js, there is a "dependency cycle detected" which prevents the app from running. Not exactly sure why this is - I suspect it might be related to the new serve.js functionality of the app.

Reply:
In utils.js you can use the exports for mediaAssets and characters from experimentHelpers.js. Then you can delete preload.js